src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -364,7 +364,7 @@ private protected virtual VBuffer<float> InitializeWeightsSgd(IChannel ch, Float
             return result;
         }
 
-        protected abstract VBuffer<float> InitializeWeightsFromPredictor(TModel srcPredictor);
+        protected abstract VBuffer<float> InitializeWeightsFromPredictor(IPredictor srcPredictor);
```

Comment:
I guess we're trading type safety on the private surface for type safety on the public surface. 😄 That's a good exchange I suppose.